book/glossary.md

```diff
@@ -207,8 +271,39 @@ pixels.
 
 __Visual effect__: A CSS property that does not affect __layout__.
 
+__Zoom__: Changing the ratio of CSS sizes to pixels in order to  make
+content on a web page larger or smaller.
+
 Computer technologies
 =====================
 
+__Assistive technology__: Computer software used to assist people in using
+the computer or web browser. The most common are screen readers.
+
+__CPU__: Centra Processing Unit, a generic piece of hardware that is able
+to excute computer programs.
```

Comment:
```suggestion
__CPU__: Central Processing Unit, the hardware component in a computer
that executes generic computer programs.
```

Rewrote the definition a bit so that GPUs don't satisfy the definition.